README.md

```diff
@@ -1,5 +1,12 @@
 # eco-goinfra
 
+## Overview
+The [eco-goinfra](https://github.com/openshift-kni/eco-goinfra) is a subset of generic packages stored under public pkg 
```

Comment:
I think it's clearer to just say something like "The eco-goinfra project contains a collection of generic [packages](./pkg) that can be used across various test projects."